source/MaterialXFormat/File.cpp

```diff
@@ -137,10 +138,15 @@ FilePath FilePath::getNormalized() const
     FilePath res;
     for (const string& str : _vec)
     {
-        if (!res.isEmpty() && res[res.size() - 1] == DOT_STRING && str == DOT_STRING)
+        if (str == CURRENT_PATH_STRING)
         {
             continue;
         }
+        if (str == PARENT_PATH_STRING && !res.isEmpty() && res[res.size() - 1] != PARENT_PATH_STRING)
```

Comment:
Looks great. Thanks for adding this in. Should help with deployments which don't like ".." specifies. Believe at least I think it will help MDL.